### ISSUE TITLE
TypeError: Cannot read the property 'stack' of null - v0.5.0-alpha1

### ISSUE THREAD

**Issue by wbarrantes** [niklasvh/html2canvas#575]
I was using html2canvas v0.4.1 but I updated to v0.5.0-alpha1 to **trying to solved an issue on IE11.** (basically the canvas was cut off on IE11, works fine on Chrome and FF).

But know I get the following error (in any browser): 

![screen shot 2015-04-22 at 2 22 02 pm](https://cloud.githubusercontent.com/assets/636027/7284459/ff87dc22-e8fa-11e4-817d-666048d5735b.png)

*Uncaught (in promise) TypeError: Cannot read property 'stack' of null {stack: (...), message: "Cannot read property 'stack' of null"}message: "Cannot read property 'stack' of null"stack: (...)get stack: function () { [native code] }set stack: function () { [native code] }__proto__: Error(anonymous function) @ html2canvas.js:2092NodeParser.createStackingContexts @ html2canvas.js:2086(anonymous function) @ html2canvas.js:1932(anonymous function) @ html2canvas.js:2697*

The error in on the line html2canvas.js:2092
```javascript
container.assignStack(container.parent.stack);
```

Any thoughts?
Thanks!

**Comment by BenBBear** [comment]
I encounter the same issue, how do you fix it?

**Comment by wbarrantes** [comment]
Hi,

I'm using on the same project a game framework called 'Enchant.js'. I
noticed that framework renames a class called 'Node' ('Node' is the named
of a class used by the library html2canvas v0.5.0 and higher). To fix this
issue I've renamed the class 'Node' on enchant.js to avoid override this
class.

Checked if you have another 3-party libraries on your project, and see if
they used a class called 'Node' (or maybe is overriding another class).

Hope this helps!
Walter.

2015-05-09 8:12 GMT-06:00 Xinyu Zhang <notifications@github.com>:

> I encounter the same issue, how do you fix it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/niklasvh/html2canvas/issues/575#issuecomment-100494211>
> .
>



-- 
Walter Barrantes Fuentes
Cel. 8664-1423
